client/consensus/aura/src/lib.rs

```diff
@@ -212,6 +219,7 @@ impl<B, C, E, I, P, Error, SO> sc_consensus_slots::SimpleSlotWorker<B> for AuraW
 	type Proposer = E::Proposer;
 	type Claim = P;
```

Comment:
You just need to change this to:
```suggestion
	type Claim = P::Public;
```